### DISCUSSION TITLE
Images not rendering through Yazi

### DISCUSSION THREAD

**Discussion by AsternA** [ImageMagick/ImageMagick#7558] [Help]
### ImageMagick version

7.1.1-36

### Operating system, version and so on

MacOS, 13.6.3, Yazi (0.3.0)

### Description

Hey Guys, 
I hope I am posting this in the right place.
I have installed Yazi through Homebrew on my Mac and was scrolling trough my home directory wanting to look at the images.
When I opened the first image a system window popped up saying do you grant permission (or something of that sort) I clicked yes and the image appeared. However, the next image the same window popped up and I clicked "don't ask me again" but by mistake I clicked on "NO" and now when ever I select Images / pdfs I see binary gibberish.

I have tried to uninstall Yazi (with all the associated apps) and reinstall them but the permission window does not pop back up and all I am left with is binary garbage.

Is there a way to clear the image permission cache? So it will ask me again what I prefer to do?
(Note: I have my terminal set to Full Disk Access)
Thanks

**Comment by fmw42** [comment]
This does not sound like an Imagemagick issue, but one from Yazi.  I suggest that you contact the Yazi team about that.  As far as I know, Imagemagick does not open a window. It is a terminal command line tool. However, tool such as Yazi(?) make calls to Imagemagick.  So Yazi is in control and does the windowing.

**Reply by fmw42** [reply]
Are you sure that Yazi is using Imagemagick for displaying images and not some other tool?  Sorry, I know nothing about Yazi other than it is a terminal file manager.

**Reply by AsternA** [reply]
I think you're right that this has nothing to do with Imagemagick. Im on a different thread with Yazi repo trying to sort this mess.
Thanks for you reply but I think we can close this thread.
